pages/tutorials/3.0/system/time.md

```diff
@@ -33,9 +38,17 @@ sf::Int32 msec = time.asMilliseconds();
 float     sec  = time.asSeconds();
 ```
 
+Conversions with the C++ standard library's `std::chrono::duration` are supported in the form of a new constructor template as well as a conversion operator.
+
+```cpp
+sf::Time time = std::chrono::milliseconds(100);
```

Comment:
Maybe it should mention implicit conversion, instead of just the constructor.

Reply:
`as well as a conversion operator.`

I do mention the implicit conversion operator. Do you want it rephrased in some way?

Reply:
I was talking about the implicit construction. Basically, how would you explain to someone “junior” what that line does. Since it’s an assignment, it’s not really clear how the `sf::Time` is implicitly created, unless you really understand what “new constructor template” implies and how C++ works.